.dockerignore

```diff
@@ -1,8 +1,6 @@
-# Ignore everything except selected directory
-**
-
-!base_build
-base_build/README.md
-
-!scripts
-!scenario_tests/utils/requirements.txt
\ No newline at end of file
+.github
+.venv
+venv
+.docs
+sample
+.env
```

Comment:
This is the real problem.
For some reason we can't ignore everything and then allow selected directory like what we did in desktop/local.